internal/run/run_test.go

```diff
@@ -60,6 +61,7 @@ func TestStart(t *testing.T) {
 		firstGetConfigHashError     error
 		secondGetConfigHashResult   hash.Hash32
 		secondGetConfigHashError    error
+		startTunnelCallCount        int
```

Comment:
ain't daemon suppose to have only one tunnel to load balancer?

Reply:
Yep this is just to test if the function is called (0 or 1)